Clients/GUI/Pages/BotEdition.razor

```diff
@@ -0,0 +1,201 @@
+                    <br />
+                }
+            </WizardStep>
+            <WizardStep Name="Bot rules">
+                <br>
+                <div>
+                    @foreach (var rule in initialRules)
+                    {
+                        <div class="form-check">
+                            <br>
+                            <input class="form-check-input" type="checkbox" @bind-value="@rule.IsChoosen">
+                            <label class="form-check-label">@rule.Description</label>
+                            @if (rule.IsChoosen)
+                            {
+                                <div>
+                                    <label>Parameters:</label>
+                                    <div class="form-row input-group-prepend">
+                                        <br>
+                                        <label class="col-8 ">Percentage of the user's balance</label>
+                                        <input @bind-value="@rule.MoneyLimitPercents" type="number" class="col-4">
+                                    </div>
+                                    <div class="form-row input-group-prepend">
+                                        <label class="col-8 ">Trigger value to trade</label>
+                                        <input @bind-value="@rule.TriggerValue" type="number" class="col-4">
+                                    </div>
+                                    <div class="form-row input-group-prepend">
+                                        <label class="col-8 ">Time interval</label>
+                                        <input @bind-value="@rule.TimeMarker" type="@(@rule.IsChoosen ? "time" : "hidden")" class="col-4">
+                                    </div>
+                                </div>
+                                <br>
+                            }
+
+                        </div>
+                    }
+                </div>
+                <ValidationSummary />
+
+                @if (errorServerMessage.Length != 0)
+                {
+                    <label style="color: red;"> @errorServerMessage </label>
+                    <br />
+                }
+                @if (showLoading)
+                {
+                    <div class="spinner-border" role="status">
+                        <span class="sr-only">Loading...</span>
+                    </div>
+                }
+                <br>
+            </WizardStep>
+        </Wizard>
+    </EditForm>
+</div>
+
+@code {
+    [CascadingParameter] BlazoredModalInstance BlazoredModal { get; set; }
+    [Parameter] public BotData CurrentBot { get; set; }
+    private BotModel userInput;
+    private EditContext editContext;
+    private bool showLoading = false;
+    private string errorServerMessage = "";
+    private List<BotRuleModel> initialRules;
+    private UserToken userToken;
+
+    private void InitializeBotRules()
+    {
+        BotRuleModel ruleBuy = null;
```

Comment:
Why not to unite buy and sell options in one rule?

Reply:
They have little different descriptions. And I think that it could be important to have the opportunity for person to select two different parameters for sell and buy